stdlib/contrib/jsternberg/aggregate/aggregate.flux

```diff
@@ -6,38 +7,47 @@ import "contrib/jsternberg/math"
 // table will aggregate columns and create tables with a single
 // row containing the aggregated value.
 //
-// This function takes a single parameter of `columns`. The parameter
-// is an object with the output column name as the key and the aggregate
-// object as the value.
+// ## Parameters
+// - tables: Input data. Default is piped-forward data (`<-`).
+// - columns: Columns to aggregate and which aggregate method to use.
+//      Columns is a record where the key is a column name and the value is an aggregate record.
+//      The aggregate record is composed of at least the following required attributes:
+//          column = string
+//              The column name for the input.
+//          init = (values) -> state
+//              An initial function to compute the initial state of the
+//              output. This can return either the final aggregate or a
+//              temporary state object that can be used to compute the
+//              final aggregate. The values parameter will always be a
+//              non-empty array of values from the specified column.
+//          reduce = (values, state) -> state
+//              A function that takes in another buffer of values
+//              and the current state of the aggregate and computes
+//              the updated state.
+//          compute = (state) -> value
+//              A function that takes the state and computes the final
+//              aggregate.
+//          fill = value
+//              The value passed to fill, if present, will determine what
+//              the aggregate does when there are no values.
+//              This can either be a value or one of the predefined
+//              identifiers of null or none.
+//              This value must be the same type as the value return from
+//              compute.
```

Comment:
```suggestion
//          - **column**: Input column name (string).
//          - **init**: A function to compute the initial state of the
//              output. This can return either the final aggregate or a
//              temporary state object that can be used to compute the
//              final aggregate. The `values` parameter will always be a
//              non-empty array of values from the specified column.
//              For example: `(values) => state`.
//          - **reduce**: A function that takes in another buffer of values
//              and the current state of the aggregate and computes
//              the updated state.
//              For example: `(values, state) => state`.
//          - **compute**: A function that takes the state and computes the final
//              aggregate. For example, `(state) => value`.
//          - **fill**: The value passed to `fill()`. If present, the fill value
//              determines what the aggregate does when there are no values.
//              This can either be a value or one of the predefined
//              identifiers of `null` or `none`.
//              This value must be the same type as the value return from
//              `compute`.
```